core/src/services/UnifiedSearchService.js

```diff
@@ -112,7 +112,7 @@ export async function getContacts({ searchTerm }) {
 		let authenticatedUser = getCurrentUser()
 		authenticatedUser = {
 			id: authenticatedUser.uid,
-			fullName: 'Me',
+			fullName: t('core', 'Me'),
```

Comment:
```suggestion
			fullName: authenticatedUser.displayName,
```
I think the own display name is better? Also removes the question if it should be "me" or "myself"

Cc @jancborchardt

Reply:
Yes, generally we try to avoid using any variant of "Me" in the UI, so display name will work better :)

Reply:
![grafik](https://github.com/nextcloud/server/assets/213943/6a4d160e-151e-4fca-8f79-19cc46e191d6)